xgboost_synthetic/build-train-deploy.ipynb

```diff
@@ -27,22 +27,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Verify we have a GCP account\n",
```

Comment:
I think we should still run a check to see if google application credentials? Can you call the function get_application_default credentials? e.g.
https://github.com/kubeflow/testing/blob/16ea186441a45d8e74457317a177d232eea75a71/py/kubeflow/testing/cleanup_ci.py#L105

If that raises an exception; e.g. because we couldn't get credentials then we should print out an actionable error message.

Reply:
added